modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -147,7 +147,7 @@
 	alt_titles = list("Security Officer", "Security Operative", "Peacekeeper")
 
 /datum/job/shaft_miner
-	alt_titles = list("Shaft Miner", "Excavator", "Spelunker", "Drill Technician", "Prospector")
+	alt_titles = list("Union Miner", "Union Excavator", "Union Spelunker", "Union Drill Technician", "Union Prospector")
```

Comment:
Do they *all* need to have Union in the name, though? I feel like it'll mostly just make all the alt-titles longer and more complicated for very little gain.

Reply:
> Do they _all_ need to have Union in the name, though? I feel like it'll mostly just make all the alt-titles longer and more complicated for very little gain.

i'm talking to someone about a change for the better, that being, it'll addendum it after the title, like
John Jackson, (Shaft Miner) (FTU)

which is why it's still in draft